doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -201,6 +202,114 @@ This code is used in the sample below which implements an [SelectComponent](../s
+            InlineEditorTemplateScript="WebGridCellEditCellTemplate"
+            Editable="true"
+            Name="column1"
+            @ref="column1">
+            </IgbColumn>
+
+
+```
+
+and pass the template:
+
+```javascript
+
+igRegisterScript("WebGridChangePercentTemplate", (ctx) => {
+    let cellValues = [];
+    let uniqueValues = [];
+    for(const i of this.webGridCellEditSampleRoleplay){
+        const field = ctx.cell.column.field;
+        if(uniqueValues.indexOf(i[field]) === -1 )
+        {
+            cellValues.push(html`<igc-select-item value=${i[field]}>${(i[field])}</igc-select-item>`);
+            uniqueValues.push(i[field]);
+        }
+    }
+    return html`<div>
+    <igc-select position-strategy="fixed" @igcChange=${ e => ctx.cell.editValue = e.detail.value}>
+          ${cellValues}
+    </igc-select>
+</div>`;
+}, false);
+
+
+```
+Working sample of the above can be found here for further referencee: 
+
+`sample="/{ComponentSample}/cell-editing-sample", height="650", alt="{Platform} {ComponentTitle} Cell Editing Template Sample"`
+
+<!-- end: Blazor -->
+
+<!-- WebComponents -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. First create the column as you usually would:
+
+```html
+
+<igc-column
+            field="race"
+            header="Race"
+            data-type="string"
+            editable="true"
+            name="column1"
+            id="column1">
+            </igc-column>
+
+```
+
+and pass the templates to this column in the index.ts file:
+
+```ts
+
+constructor() {
+        var grid1 = this.grid1 = document.getElementById('grid1') as IgcGridComponent;
+        var column1 = this.column1 = document.getElementById('column1') as IgcColumnComponent;
+        var column2 = this.column2 = document.getElementById('column2') as IgcColumnComponent;
+        var column3 = this.column3 = document.getElementById('column3') as IgcColumnComponent;
+
+        grid1.data = this.webGridCellEditSampleRoleplay;
+        column1.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+        column2.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+        column3.inlineEditorTemplate = this.webGridCellEditCellTemplate;
+    }
+
+
+public webGridCellEditCellTemplate = (ctx: IgcCellTemplateContext) => {
+        let cellValues: any = [];
+        let uniqueValues: any = [];
+        for(const i of (this.webGridCellEditSampleRoleplay as any)){
+            const field: string = ctx.cell.column.field;
+            if(uniqueValues.indexOf(i[field]) === -1 )
+            {
+                cellValues.push(html`<igc-select-item value=${i[field]}>${(i[field])}</igc-select-item>`);
+                uniqueValues.push(i[field]);
+            }
+        }
+        return html`
+        <igc-select style="width:100%; height:100%" size="large" @igcChange=${(e: any) => ctx.cell.editValue = e.detail.value}>
```

Comment:
Not sure if we should show snippet/sample with select until all integration issues are resolved.